src/execution/operator/persistent/physical_export.cpp

```diff
@@ -22,7 +22,15 @@ static void WriteCatalogEntries(stringstream &ss, vector<reference<CatalogEntry>
 		if (entry.get().internal) {
 			continue;
 		}
-		ss << entry.get().ToSQL() << '\n';
+		auto create_info = entry.get().GetInfo();
+		if (create_info->HasToString()) {
```

Comment:
Can we turn `ToString` into a `NotImplementedException`, and then use a try/catch instead of `HasToString`?